api/v1/requests/AttendeeRequest.js

```diff
@@ -40,6 +40,10 @@ var bodyValidations = {
 	'attendee.linkedin':  ['required', 'string', 'maxLength:50'],
 	'attendee.interests': ['required', 'string', 'maxLength:255'],
 	'attendee.status': ['string', registration.verifyStatus],
+	'attendee.priority': ['required', 'integer', 'max:10', 'min:0'],
```

Comment:
It would be a separate method. That way you can just have them supply the relevant values and then that's it (plus it means we don't do unnecessary insertions).